sqldelight-compiler/src/test/kotlin/com/squareup/sqldelight/core/queries/MutatorQueryFunctionTest.kt

```diff
@@ -11,31 +12,31 @@ class MutatorQueryFunctionTest {
   @get:Rule val tempFolder = TemporaryFolder()
 
   @Test fun `mutator method generates proper method signature`() {
-    val file = FixtureCompiler.parseSql("""
-      |CREATE TABLE data (
-      |  id INTEGER NOT NULL PRIMARY KEY,
-      |  value TEXT AS kotlin.collections.List
-      |);
-      |
-      |insertData:
-      |INSERT INTO data
-      |VALUES (?, ?);
-      """.trimMargin(), tempFolder)
-
-    val insert = file.namedMutators.first()
-    val generator = MutatorQueryGenerator(insert)
-
-    assertThat(generator.function().toString()).isEqualTo("""
-      |override fun insertData(id: kotlin.Long?, value: kotlin.collections.List?) {
-      |  driver.execute(${insert.id}, ""${'"'}
-      |  |INSERT INTO data
-      |  |VALUES (?, ?)
-      |  ""${'"'}.trimMargin(), 2) {
-      |    bindLong(1, id)
-      |    bindString(2, if (value == null) null else database.dataAdapter.valueAdapter.encode(value))
-      |  }
-      |}
-      |""".trimMargin())
+    for (dialectPreset in listOf(DialectPreset.SQLITE_3_18, DialectPreset.MYSQL)) {
```

Comment:
can we use parameterized tests for this? would like the test failure to make it obvious if its mysql/sqlite

Reply:
There's currently an issue where it generates the MySQL id column to a `Int?` instead of a `Long`. Once I fix that, I should be able to parameterize it. I could also just duplicate the test for now if we want.

Reply:
got it, this is fine for me now